landing-pages/site/static/.htaccess

```diff
@@ -1,6 +1,8 @@
-RedirectMatch Permanent ^/docs/(stable|1.10.10)/api(\.html)?$ "https://airflow.apache.org/docs/$1/rest-api-ref"
-RedirectMatch Permanent ^/docs/(stable|1.10.10)/(cli|macros)(\.html)?$ "https://airflow.apache.org/docs/$1/$2-ref"
+RedirectMatch Permanent ^/docs/(stable|1.10.10)/api(\.html)?$ "https://airflow.apache.org/docs/apache-airflow/$1/rest-api-ref"
```

Comment:
This is an important change, but unfortunately Github does not make it very clear.

<img width="1440" alt="Screenshot 2020-12-03 at 15 40 20" src="https://user-images.githubusercontent.com/12058428/101042828-dffcc180-357d-11eb-87a6-68780fcc1a2f.png">